capstone/src/test/java/com/google/step/clients/MetricsClientTests.java

```diff
@@ -0,0 +1,363 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.step.clients;
+
+import static com.google.appengine.api.datastore.FetchOptions.Builder.withLimit;
+
+import com.google.appengine.api.datastore.*;
+import com.google.appengine.api.datastore.Query.FilterPredicate;
+import com.google.appengine.api.datastore.Query.CompositeFilter;
+import java.util.*;
+import static org.junit.Assert.*;
+import org.junit.*;
+
+
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+
+import java.util.Calendar;
+
+public class MetricsClientTests {
+
+    private final LocalServiceTestHelper helper = new LocalServiceTestHelper(new LocalDatastoreServiceTestConfig());
+    private final DatastoreService datastoreService = DatastoreServiceFactory.getDatastoreService();
+    private final MetricsClient metricsClient = new MetricsClient();
+
+    @Before
+    public void setUp() {
+        helper.setUp();
+    }
+
+    @After
+    public void tearDown() {
+        helper.tearDown();
+    }
+
+    public void checkIfDataStoreWorks() {
+        assertEquals(0, datastoreService.prepare(new Query("User")).countEntities(withLimit(10)));
+        datastoreService.put(new Entity("User"));
+        datastoreService.put(new Entity("User"));
+        assertEquals(2, datastoreService.prepare(new Query("User")).countEntities(withLimit(10)));
+    }
+
+    @Test
+    public void testCheckWorks1() {
+        checkIfDataStoreWorks();
+    }
+
+    @Test
+    public void testCheckWorks2() {
+        checkIfDataStoreWorks();
```

Comment:
Personally, I like to keep them just in case we change something from the test class and something fails, in order to make sure its not something datastore related.